CMakeLists.txt

```diff
@@ -109,42 +103,8 @@ if(BUILD_BENCHMARKS)
   add_subdirectory(benchmarks)
 endif(BUILD_BENCHMARKS)
 
-###################################################################################################
-# - library targets -------------------------------------------------------------------------------
-
-add_library(rmm SHARED "${CNMEM_SOURCE_DIR}/cnmem.cpp")
```

Comment:
The library target should remain. `rmm` is still a library, just a header-only one. For that, use

```
add_library(rmm INTERFACE)
```
as it has no sources.

Reply:
Done.